src/game/activemenutick.c

```diff
@@ -337,6 +359,12 @@ void amTick(void)
 				}
 			}
 		}
+#ifndef PLATFORM_N64
+        else {
+            g_AmMenus[g_AmIndex].mousex = 0.f;
+            g_AmMenus[g_AmIndex].mousey = 0.f;
+        }
+#endif
```

Comment:
This code is also not indented the same as the rest of the code. This file uses tabs put your editor is inserting spaces.